system/lib/wasmfs/file.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+// This file defines the file object of the new file system.
+// Current Status: Work in Progress.
+// See https://github.com/emscripten-core/emscripten/issues/15041.
+
+#include "file.h"
+#include <emscripten/html5.h>
+#include <vector>
+
+static std::vector<char> fdWriteStdoutBuffer;
+static std::vector<char> fdWriteStderrBuffer;
```

Comment:
Can we make these private within the relevant `File` subclasses to ensure they are not unsafely accessed by any other code?